pom.xml

```diff
@@ -372,6 +360,18 @@
   </dependencyManagement>
 
   <dependencies>
+    <!-- all modules use slf4j-api -->
+    <dependency>
```

Comment:
not all. we can't do this here.. this will add the dependency to zipkin's core jar which will end up everywhere in brave etc. I think you'll want to move this to dependency management